src/Config/FileLoader.php

```diff
@@ -183,35 +186,33 @@ protected function getPackagePath($package, $group, $env = null)
 
         if (!$env) {
             $file = "{$package}/{$group}.php";
-        }
-        else {
+        } else {
             $file = "{$package}/{$env}/{$group}.php";
         }
 
-        return $this->defaultPath.'/'.$file;
+        return $this->defaultPath . '/' . $file;
     }
 
     /**
      * Get the configuration path for a namespace.
      *
-     * @param  string  $namespace
-     * @return string
+     * @param string $namespace
+     * @return string|void
```

Comment:
When would this return void?

Reply:
If the elseif is not true

Reply:
What conditions would trigger that? I'm wondering if we should have better handling for it than just returning void

Reply:
If the namespace doesn't exist in $this->hints. How would you like me to solve it? add a return null to the bottom?

Reply:
> If the namespace doesn't exist in $this->hints

I know that 😛 

I'm curious to know what application state would cause that (as well as what happens if it does occur), and if it's at risk of happening in the course of normal operations (or even non-normal operations) should we be considering throwing an exception instead?

Reply:
@LukeTowers `return null` is the best option, see line 123. After `$path = $this->getPath($namespace)` an `is_null` check is done. `is_null` returns true voor type void. So it won't break any functionality :)

Reply:
Sounds good, lets make the switch then

Reply:
@LukeTowers done :)